setup.py

```diff
@@ -6,11 +6,13 @@
 import platform
 from multiprocessing import cpu_count
 from pathlib import Path
+import site
 import sys
 import os
+import re
 
 PARENT = Path(__file__).parent.absolute()
-PYENV_PREFIX = Path("/".join(sys.executable.split('/')[:-2]))
+PYENV_PREFIX = Path(site.PREFIXES[0])
```

Comment:
@SanderHulst What is the return value of `site.PREFIXES` in CentOS version you're using? I agree it's better to use `site` if we want to find site packages. However it's usually `lib/site-packages` or `lib/pythonX.Y/site-packages` that we put some library under, so it's a little bit meaningless to put `lib` under the `site-packages` again (as in line 68 below). I'm not sure what the thinking was behind the `PYENV_PREFIX` but I guess it was mostly related to finding shared libraries location rather than python `site-packages`.